xlml/utils/aotc_reproducibility.py

```diff
@@ -0,0 +1,98 @@
+"Bash helper commands for AOTC artifacts"
```

Comment:
If this your team's specific helper functions. We usually put it under team's folder.

Reply:
moved